docs/learn/welcome.md

```diff
@@ -26,7 +26,7 @@ Connect your app to Ceramic using a [**development framework**](../build/framewo
 
 ### **Operate the Ceramic network**
 
-Provide storage, compute, and bandwidth to the network by [**running a node**](../run/index.md).
+Ceramic Provides storage computation and bandwidth to the network by [**running a node**](../run/index.md).
```

Comment:
Why change that? It was in imperative mood before, now it is indicative, and have lost any notion of an action.